### PR TITLE
Add ability to change the order of downloads and ignore episodes

### PR DESCRIPTION
**Pull request by jaischeema**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ If you want to limit the download to e.g. the last 3 episodes, do:
 
     ./sipper --user me@example.com --pw mypassword --max 3
 
+If you want to change the order of downloading from first to last, do:
+
+    ./sipper --user me@example.com --pw mypassword --asc
```
Comment (henrik) [inline README.md]: Hm, maybe `--oldest-first` as in @zeroum's PR #15? Could be a clearer name.

Love that the PR comes with docs :)

File: lib/sipper/cli.ex [hunk 2]
```diff
@@ -23,6 +23,8 @@ defmodule Sipper.CLI do
       switches: [
         max: :integer,
         dir: :string,
+        ignore: :string,
+        asc: :boolean
```
Comment (henrik) [inline lib/sipper/cli.ex]: Would you mind putting a trailing comma on this one as well? It's [my preferred style](https://github.com/barsoom/devbook/tree/master/styleguide#add-commas-to-the-end-of-multiline-lists-and-hashes).

**Comment by henrik** [comment]
Thanks so much for this! Looks awesome. I'll write some review comments.

The sorting thing was also in #15 but I haven't heard back from @zeroum.

File: lib/sipper/runner.ex [hunk 3]
```diff
@@ -14,6 +16,19 @@ defmodule Sipper.Runner do
...
+      Enum.all?(ignore_episodes, fn x -> !String.contains?(title, x) end)
+    end)
+  end
+
+  defp change_download_order(episodes, ascending) do
+    cond do
+      ascending -> Enum.reverse(episodes)
+      true -> episodes
+    end
+  end
```
Comment (henrik) [inline lib/sipper/runner.ex]: How do you feel about pattern matching on method signatures instead here? E.g. `defp change_download_order(episodes, true), do: …`

Would let us get rid of the `cond`.

**Comment by zeroum** [comment]
> Thanks so much for this! Looks awesome. I'll write some review comments.
> The sorting thing was also in #15 but I haven't heard back from @zeroum.

Sorry, I´ve got in a nasty crunch time for a project, I haven't been able to surface back to real life since then :-) Be back in a week.

File: README.md (hunk 1 above)
Comment (jaischeema) [inline README.md]: Will do :+1:

File: lib/sipper/runner.ex (hunk 3 above)
Comment (jaischeema) [inline lib/sipper/runner.ex]: Sounds good, this is my first dip into any elixir code :smile:

**Comment by jaischeema** [comment]
@henrik Thanks for reviewing this, I am still figuring out the best ways to write Elixir code and would welcome all the feedback. :smile:

**Comment by henrik** [comment]
@jaischeema Hehe, yeah, this was one of my first Elixir projects :) Very glad if it can be a learning project for others, too. Off to bed soon but I hope to look at this PR again in a day or two!

**Comment by cars10w** [comment]
heh, I am glad I checked the pull requests. Otherwise I would have created the third implementation of the --oldest-first feature. ,-)

**Comment by henrik** [comment]
Alright, merged. Thank you so much, @jaischeema!

And @cars10w :D Now it's in!

**Comment by jaischeema** [comment]
Thank you @henrik :+1:

**Pull request merged**